Dockerfile

```diff
@@ -8,6 +8,7 @@ ARG jar_name=picard.jar
 RUN apt-get update && \
     apt-get --no-install-recommends install -y --force-yes \
         git \
+	r-base \
```

Comment:
use spaces, not tabs, to indent lines

Reply:
I have added a second commit that uses spaces instead of tabs.